samples/subsys/usb_c/source/src/power_ctrl.c

```diff
@@ -113,7 +113,7 @@ static int power_ctrl_init(void)
 		return -ENOENT;
 	}
 
-	if (!device_is_ready(pwm_ctl.dev)) {
+	if (!pwm_is_ready_dt(&pwm_ctl)) {
 		LOG_ERR("Error: PWM CTL device is not ready\n");
 		return -ENOENT;
```

Comment:
While at it, you could change it to `ENODEV` which is more suitable here. Otherwise LGTM.

Reply:
Thanks, I've fixed up that one and few others and I also noticed there was unnecessary newlines in logging use.